app/controllers/users_controller.rb

```diff
@@ -18,10 +21,36 @@ def create
     end
   end
 
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update_attributes(user_params)
+      flash[:success] = 'Profile updated'
+      redirect_to @user
+    else
+      render 'edit'
+    end
+  end
+
   private
 
     def user_params
       params.require(:user).permit(:name, :email, :password,
                                    :password_confirmation)
     end
+
+    def logged_in_user
+      unless logged_in?
+        flash[:danger] = 'Please log in.'
+        redirect_to login_url
+      end
+    end
+
+    def correct_user
```

Comment:
__[reek]__
* [UnusedPrivateMethod] has the unused private instance method `correct_user` [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Private-Method.md)